### ISSUE TITLE
Right click on bookmarks opens 'mydfltmenu'

### ISSUE THREAD

**Issue by WernerFP** [aaFn/Bookmark-search-plus-2#92]
<p align="center">
<img src="https://user-images.githubusercontent.com/22373662/55276552-3ae40d00-52f5-11e9-918e-b1045e54da4f.jpg"></img>
</p>

`panel.js` line 5142:
```
// Right click on body or on the magnifier glass opens mydfltmenu
```

Right click on body doesn't trigger `mydfltmenu` (in my Linux systems) but right click on bookmarks opens the menu unintentionally. The desired menu `mybbkmkmenu` is displayed for milliseconds and then replaced by the menu `mydfltmenu`. This issue occurs (reproducible in my case) as mouseup event:
By holding down the right mouse button the menu `mybbkmkmenu` remains open and can be selected. However, if there is a bookmark behind the selected menu command, the mouseup event does not execute the selected command, but triggers the menu `mydfltmenu` above this bookmark.

To be able to use the context menu (`mybbkmkmenu`) I often need 5 right clicks or more to make it work.

(_I noticed this issue after updating to Firefox 66. But I'm not sure if this is just a coincidence._)

**Comment by aaFn** [comment]
Hello @WernerFP , yes this is a bug of FF66 on Linux only apparently .. :-(

I noticed it also myself and analyzed to find the root cause.

It is due to a target element change in the click event subsequent to a contextmenu event.
Apparently it is now always set to body (which is the place for the default menu, hence it appearing ..).

I made a workaround by:
- remembering the target element at mousedown event time
- then comparing it with the target at click event time, and if different, ignore the target in the click event so as to not open the default menu.

I then modified the behavior of the default menu to attach it only to the magnifier glass, for safety and also in preparation for #90

It will be in 2.0.58 tonight.
Thanks, aaFn.
